### PR TITLE
open dendron from terminal

### PR DESCRIPTION
**Pull request by Ding-Fan**
## What issues does this PR fix or reference?

- Fixes:
OR
- Relates to:

> **NOTE:** `Fixes` will close the ticket after merging. `Relates to:` will leave the ticket open after merging.

### Merge requirements satisfied

> For more information, visit the [Contributing Guide for Documentation](https://wiki.dendron.so/notes/b58801fc-43a9-4d42-a58b-eabc3e8538cb/)

- [ ] Check rendered output to ensure formatting is correct for renderings of wikilinks, note refs, tables, and images. `Dendron: Show Preview` can be used in your workspace to confirm the page renders as expected. Sometimes, `Dendron: Reload Index` needs to be ran if certain wikilinks aren't working as expected. (ignore this if contribution is made via the `Edit this page on GitHub` button from the published site)
- [ ] If this PR includes any refactoring (renaming notes, renaming headers, etc.), make sure that these changes were done via [Dendron Refactoring Commands](https://wiki.dendron.so/notes/srajljj10V2dl19nCSFiC/). This will ensure that any other impacted areas of the documentation, that link to or embed modified notes (via note references), are automatically updated.

Verify GitHub Actions tests are passing, which can be seen in the `Checks` tab of the PR:

- [ ] `URL validator` GitHub Action passes
- [ ] `Dendron site build` GitHub Action passes

> **NOTE:** If you are a new contributor, a maintainer will need to provide approval for GitHub Actions to run on your PRs.

### CODE REVIEW

File: vault/dendron.guides.tips.md [hunk 1]
```diff
@@ -117,6 +117,15 @@ You can streamline this process by assigning a keyboard shortcut to the command.
 
 First copy it into Notion or use the Notion web clipper to clip it. This will format it nicely in Markdown. You can then paste it into Dendron with Markdown and everything :)
 
+### Open Dendron from terminal
+
```
Comment (flammehawk) [inline vault/dendron.guides.tips.md:121]: I would add a Header for the Os this is use full for here

```suggestion

#### Linux & Mac & WSL

```

File: vault/dendron.guides.tips.md [hunk 2]
```diff
@@ -117,6 +117,15 @@ You can streamline this process by assigning a keyboard shortcut to the command.
...
+You can create an alias in your `.bashrc` / `.zshrc` file. Remember to replace `path/to/your/` with your `code-workspace` file path.
+
+```sh
+alias dendron="code path/to/your/dendron.code-workspace"
+```
+
```
Comment (flammehawk) [inline vault/dendron.guides.tips.md:127]: Quick suggestion for Similiar effects with Windows Command Prompt and Powershell

````suggestion

##### Windows

###### CMD

To achieve something similar for the CMD prompt of windows you can do the following: 

Create a `alias.txt`(for example in your user dir):

```txt

dendron=code {path/to/your/dendron.code-workspace}

``` 

Then you can do the following

```cmd
doskey /macrofile=C:\Users\example\alias.txt
``` 

After doing that dendron becomes available in the cmd prompt. 
To automate this you can take a look here: [Setting Init script for Windows Command Prompt](https://sukhbinder.wordpress.com/2021/06/16/setting-init-script-for-windows-command-prompt/)

#### All

##### Powershell

For powershell you add the following to your Powershell `$PROFILE`:
 
```ps
Set-Alias -Name dendron -Value code {path/to/your/dendron.code-workspace}
```

````

**Review by flammehawk — COMMENTED** [review]
Sorry for hijacking, but
maybe add my suggestions to it which expand your additions with tips for adding a windows section to it.

If not wanted ping me and I will create a new PR for these changes.

File: vault/dendron.guides.tips.md (hunk 2 above)
Comment (Ding-Fan) [reply vault/dendron.guides.tips.md:129]: The header level looks unclear, maybe `Windows` should have the same level as `Linux & Mac & WSL` ? 
I'm not familiar with Windows OS nowadays, why the header `All` before `Powershell` ?

Comment (flammehawk) [reply vault/dendron.guides.tips.md:129]: Yeah, that header level should be the same.

Powershell runs on every OS that's why it is under all and not under Windows.

**Comment by Ding-Fan** [comment]
Is there anything I can do with the run brok failure :thinking:

**Comment by kevinslin** [comment]
we need to fix the brok validator. meanwhile, this is great. thanks for the PR!

**Pull request merged**